### PR TITLE
[C-537] Add button corner prop, text inherit color

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

- Adds `corner` prop to button
- Adds `inherit` option for `color` prop for text

image

### Dragons

is `corner` the prop type we want here? the goal is to have a "variant" type option for pill vs rounded buttons.

### CODE REVIEW

File: packages/mobile/src/components/core/Button.tsx [hunk 1]
```diff
@@ -293,6 +317,8 @@ export const Button = (props: ButtonProps) => {
 
   const PressableComponent = url ? Link : Pressable
 
+  console.log(styles.root)
```
Comment (sliptype) [inline packages/mobile/src/components/core/Button.tsx:320]: log

**Review by sliptype — APPROVED** [review]
Do we want to support different types of corners for each variant? Or should `pill` just be a variant

**Comment by dylanjeffers** [comment]
Yeah that's what I'm not sure about. Julian mentioned in the figma comment that there is a rounded and pill style, so felt like a different category like size, ya know?

**Comment by nicoback2** [comment]
`corners: 'pill' | 'rounded'` LGTM if that's how Julian classifies it, though I think you could also just do `pill?: true | false` if the only corner variants are going to be normal (rounded) and pill forever

**Review by nicoback2 — APPROVED** [review]

**Comment by dylanjeffers** [comment]
> `corners: 'pill' | 'rounded'` LGTM if that's how Julian classifies it, though I think you could also just do `pill?: true | false` if the only corner variants are going to be normal (rounded) and pill forever

yeah i can def see pill: true/false, also not sure about "corners" as the prop, so maybe if have another variant like squared, or transparent, or shadowed, maybe update prop name to reflect that type of direction....

**Pull request merged**